docs/src/tutorials/Start.md

```diff
@@ -24,7 +24,7 @@ Before starting the Signer, you should make sure that network is disabled. Many
 
 ## First start
 
-When you first launch Signer, it prompts you to read and accept terms and conditions and privacy policy. Once that is done, the database is pre-populated with built-in networks and Signer is ready for use. It could [import network data](./New-Network.md) or [read transactions](./Kusama-tutorial.md), but to sign anything you need to create keys.
+When you first launch Signer, it prompts you to read and accept terms and conditions and privacy policy. Once that is done, the database is pre-populated with built-in networks and Signer is ready for use. It could [import network data](./Add-New-Network.md) or [read transactions](./Kusama-tutorial.md), but to sign anything you need to create keys.
```

Comment:
"for advanced users: If you would like to replace general verifier certificate (link to explanation?), this is time to do it, as that operation will wipe all data from Signer" - something like this might be useful here

Reply:
I tried to remove the certificate in parity-signer. but do not find how to add a new certificate?

Reply:
Added TODO comment. This can be done in the next docs PR